ipa.tf

```diff
@@ -548,36 +633,49 @@ secrets:
       eabHmacKey: "${jsondecode(data.vault_kv_secret_v2.zerossl_data.data_json)["EAB_HMAC_KEY"]}"
 
 ${local.dns_configuration_values}
+alternate-external-dns:
+  image:
+    repository: ${var.image_registry}/registry.k8s.io/external-dns/external-dns
 external-dns:
   enabled: ${local.enable_external_dns}
+  image:
+    repository: ${var.image_registry}/registry.k8s.io/external-dns/external-dns
   logLevel: debug
   policy: sync
-  txtOwnerId: "${var.label}-${var.region}"
+  txtOwnerId: "${local.dns_name}"
   domainFilters:
-    - ${lower(var.aws_account)}.indico.io.
+    - ${local.dns_zone_name}
 
   provider: aws
-  aws:
-    zoneType: public
-    region: ${var.region}
-
   policy: sync
   sources:
     - service
     - ingress
 aws-for-fluent-bit:
   enabled: true
+  image:
+    repository: ${var.image_registry}/public.ecr.aws/aws-observability/aws-for-fluent-bit
   cloudWatchLogs:
     region: ${var.region}
     logGroupName: "/aws/eks/fluentbit-cloudwatch/${var.label}/logs"
     logGroupTemplate: "/aws/eks/fluentbit-cloudwatch/${var.label}/workload/$kubernetes['namespace_name']"
+ipaConfig:
+  image:
+    registry: ${var.image_registry}
+rabbitmq:
+  rabbitmq:
+    image:
+      registry: ${var.image_registry}
 cluster-autoscaler:
   cluster-autoscaler:
     awsRegion: ${var.region}
     image:
+      repository: ${var.image_registry}/public-gcr-k8s-proxy/autoscaling/cluster-autoscaler
       tag: "v1.20.0"
```

Comment:
Do we still need to hard-code this tag?

Reply:
That wasn't part of my change. We probably could remove it.

Reply:
Yeah, I just noticed it as I was reading the changes, doesn't seem like something we should manage here